tests/infra/remote.py

```diff
@@ -260,7 +262,8 @@ def start(self):
         cmd = self._cmd()
         LOG.info("[{}] {}".format(self.hostname, cmd))
         self.client.exec_command(cmd, get_pty=True)
-        self.pid()
+        if self.is_ccf_node:
```

Comment:
Our remotes used to work fine for both nodes and client until the recent fact that we rely on the node outputing its PID in a file.

Reply:
FSVO fine, we were grepping ps to decide what to suspend :)